### DISCUSSION TITLE
Logging option?

### DISCUSSION THREAD

**Discussion by fabianrinker** [moneymanagerex/moneymanagerex#5327] [Q&A] [answered]
Hi,
when I lately experienced #5318, which I was not able to reproduce the second day, I wondered if there is some kind of logging also in case no crash happens?

I imagine some (debug) tracing like
```
[Info] Created new transaction
[Debug] Account view XYZ
[Warning] WebApp not accessible
...
```

Is there already some logging which could be switched on?

**Comment by whalley** [comment, marked as answer]
Nope. There is some minimal logging when built in debug mode but that is not released and only used by developers.
